em-client/src/client/mod.rs

```diff
@@ -6531,3 +6522,89 @@ impl error::Error for ClientInitError {
         "Failed to produce a hyper client."
     }
 }
+
+fn deserialize_config_checked(raw_config: &str, expected_hash: &[u8; 32]) -> Result<models::RuntimeAppConfig, ApiError> {
```

Comment:
Use constant SHA256 digest length instead of 32 here. And follow the same in other parts of this PR.